protocol/x/vault/keeper/params_test.go

```diff
@@ -44,37 +43,51 @@ func TestGetSetParams(t *testing.T) {
 		OrderExpirationSeconds:           0, // invalid
 		ActivationThresholdQuoteQuantums: dtypes.NewInt(1_000_000_000),
 	}
-	err = k.SetParams(ctx, invalidParams)
+	err = k.SetDefaultQuotingParams(ctx, invalidParams)
 	require.Error(t, err)
-	require.Equal(t, newParams, k.GetParams(ctx))
+	require.Equal(t, newParams, k.GetDefaultQuotingParams(ctx))
 }
 
-func TestGetSetVaultParams(t *testing.T) {
+func TestGetSetVaultQuotingParams(t *testing.T) {
```

Comment:
nit: Given the paramaterized nature of the tests now, prefer test cases pattern.